WhaleSpotting/Services/SightingsService.cs

```diff
@@ -36,6 +36,24 @@ public async Task<List<SightingResponseModel>> GetSightings()
             return sightings;
         }
 
+        public List<SightingResponseModel> CreateSightings(List<SightingDbModel> sightingsToAdd)
+        {
+            var newSightingIds = sightingsToAdd.Select(s => s.ApiId).Distinct();
+           
+            var sightingsInDbIds = _context.Sightings
+                                   .Where(s => newSightingIds.Contains(s.ApiId))
```

Comment:
These are a bit too far indented I think, should just be 1 tab

Reply:
:)